src/analysis/analytic/composition.lean

```diff
@@ -573,7 +573,7 @@ def comp_partial_sum_target_set (m M N : ℕ) : set (Σ n, composition n) :=
 
 lemma comp_partial_sum_target_subset_image_comp_partial_sum_source
   (m M N : ℕ) (i : Σ n, composition n) (hi : i ∈ comp_partial_sum_target_set m M N) :
-  ∃ j (hj : j ∈ comp_partial_sum_source m M N), i = comp_change_of_variables m M N j hj :=
+  ∃ j (hj : j ∈ comp_partial_sum_source m M N), comp_change_of_variables m M N j hj = i :=
```

Comment:
Reversign the direction here seems similarly undesirable, as `obtain \<j, hj, rfl\> := comp_partial_sum_target_subset_image_comp_partial_sum_source _ _ _ i _,` no longer works.